src/app/legacy/containers/CpsVaried/index.jsx

```diff
@@ -0,0 +1,24 @@
+import pathOr from 'ramda/src/pathOr';
+import useOptimizelyMvtVariation from '../../../hooks/useOptimizelyMvtVariation';
+
+// 004_brasil_recommendations_experiment
+const optimizelyMappings = {
+  content_recs: 'datalabContentRecommendations',
+  hybrid_recs: 'datalabHybridRecommendations',
+};
+
+const CpsVaried = ({ pageData, renderFunction, experimentID }) => {
+  let unirecsHybridRecommendationData = null;
+  const variation = useOptimizelyMvtVariation(experimentID);
+
+  if (variation && variation !== 'control') {
+    unirecsHybridRecommendationData = pathOr(
+      null,
+      [optimizelyMappings[variation]],
+      pageData,
+    );
+  }
+  return renderFunction(unirecsHybridRecommendationData);
+};
```

Comment:
Probably need some tear down of this code, or make it more generic as it seems to be specific for recommendations atm, we can do that after we've tested the A/A works